htdocs/editor_tab.js

```diff
@@ -1306,7 +1306,7 @@ var editor = function () {
                 window.open(url, "_blank");
             }
             else
-                this.load_notebook(gistname, version, source, null, selroot);
+                this.load_notebook(gistname, version, source, selroot, null);
```

Comment:
no problem - I'm still looking at some very odd behaviour in this area, so not fully ready yet.